test/stack-test-inner-loop.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+mkdir inner-loop-test-dir
+cd inner-loop-test-dir
+
+echo -e "\n> Clone application-stack-intro project"
+git clone https://github.com/OpenLiberty/application-stack-intro.git
+cd application-stack-intro
+
+echo -e "\n> Copy devfile"
+cp ../../generated/devfile.yaml devfile.yaml
+
+echo -e "\n> Create new odo project"
+odo project create inner-loop-test
+
+echo -e "\n> Create new odo component"
+odo create my-ol-component
+
+echo -e "\n> Create URL with Minikube IP"
+odo url create --host $(minikube ip).nip.io
+
+echo -e "\n> Push to Minikube"
+odo push
```

Comment:
Seems like it'd be worth adding an `odo test`.   Could you pls open up an issue on the maven repo too if we don't have one there?

Reply:
Can we open a separate issue for this maybe?